src/libs/actions/BankAccounts.ts

```diff
@@ -7,6 +7,9 @@ import * as PlaidDataProps from '../../pages/ReimbursementAccount/plaidDataPropT
 import Navigation from '../Navigation/Navigation';
 import ROUTES from '../../ROUTES';
 import * as ReimbursementAccount from './ReimbursementAccount';
+import PlaidBankAccount from '../../types/onyx/PlaidBankAccount';
+import {ACHContractStepProps, BankAccountStepProps, CompanyStepProps, OnfidoData, ReimbursementAccountProps, RequestorStepProps} from '../../types/onyx/ReimbursementAccountDraft';
```

Comment:
@rezkiy37 I've looked through the app and in most the cases it's not used

Reply:
Yeah, I see only several examples in the project. I think we should add `import type` for this particular case, because:
1. All imported elements are types. 
2. [‘import’ vs ‘import type’ in TypeScript](https://medium.com/@quizzesforyou/import-vs-import-type-in-typescript-8e5177b62bea#:~:text=%27./Shapes%27%3B-,Optimizing%20Imports%3A,-The%20import%20can).